README.rst

```diff
@@ -74,6 +74,17 @@ This command is well documented, consult the help with::
 
     ./manage.py help cities_light
 
+Development
+-----------
+
+To build the docs use the following steps:
+
+1. mkvirtualenv dcl-doc
```

Comment:
Automatic virtualenv activation is pretty cool too ;)

http://blog.yourlabs.org/post/21015702927/automatic-virtualenv-activation